lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -136,12 +136,12 @@ def create(self, trans, payload, **kwd):
             'from_history_id',
             'shared_workflow_id',
             'workflow',
-        ] ).intersection( payload )
-        if len( ways_to_create ) == 0:
+        ] )
+        if len( ways_to_create.intersection( payload ) ) == 0:
```

Comment:
Are you running workflows via this endpoint and not the POSTing to invocations? Running workflows on this endpoint has been deprecated.